Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 1087, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/anaconda3/lib/python3.7/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/anaconda3/lib/python3.7/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/opt/anaconda3/lib/python3.7/asyncio/base_events.py", line 583, in run_until_complete
    return future.result()
  File "/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 541, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 832, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 740, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Contructing heat map for better visualization
import plotly.express as px

fig = px.choropleth(locations=percap2.column('state'), locationmode="USA-states", \
                    color=percap2.column('count per cap'), scope="usa", \
                    color_continuous_scale="geyser", range_color=(10, 55))
fig.show()
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
[0;32m<ipython-input-32-e8a9a0c783a1>[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m [0;31m# Contructing heat map for better visualization[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 2[0;31m [0;32mimport[0m [0mplotly[0m[0;34m.[0m[0mexpress[0m [0;32mas[0m [0mpx[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      3[0m [0;34m[0m[0m
[1;32m      4[0m fig = px.choropleth(locations=percap2.column('state'), locationmode="USA-states", \
[1;32m      5[0m                     [0mcolor[0m[0;34m=[0m[0mpercap2[0m[0;34m.[0m[0mcolumn[0m[0;34m([0m[0;34m'count per cap'[0m[0;34m)[0m[0;34m,[0m [0mscope[0m[0;34m=[0m[0;34m"usa"[0m[0;34m,[0m[0;31m [0m[0;31m\[0m[0;34m[0m[0;34m[0m[0m

[0;31mModuleNotFoundError[0m: No module named 'plotly'
ModuleNotFoundError: No module named 'plotly'

